sdk/healthcareapis/arm-healthcareapis/CHANGELOG.md

```diff
@@ -1,15 +1,34 @@
 # Release History
+    
+## 3.0.0 (2023-12-07)
+    
+**Features**
 
-## 2.2.1 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
+  - Added Interface CorsConfiguration
+  - Added Interface Encryption
+  - Added Interface EncryptionCustomerManagedKeyEncryption
+  - Added Interface FhirServiceImportConfiguration
+  - Added Interface ImplementationGuidesConfiguration
+  - Added Interface ServiceImportConfigurationInfo
+  - Interface DicomService has a new optional parameter corsConfiguration
+  - Interface DicomService has a new optional parameter encryption
+  - Interface DicomService has a new optional parameter eventState
+  - Interface FhirService has a new optional parameter encryption
+  - Interface FhirService has a new optional parameter implementationGuidesConfiguration
+  - Interface FhirService has a new optional parameter importConfiguration
+  - Interface MetricSpecification has a new optional parameter enableRegionalMdmAccount
+  - Interface MetricSpecification has a new optional parameter isInternal
+  - Interface MetricSpecification has a new optional parameter metricFilterPattern
+  - Interface MetricSpecification has a new optional parameter resourceIdDimensionNameOverride
+  - Interface MetricSpecification has a new optional parameter sourceMdmAccount
+  - Interface ServiceCosmosDbConfigurationInfo has a new optional parameter crossTenantCmkApplicationId
+  - Interface ServicesProperties has a new optional parameter importConfiguration
 
-### Other Changes
+**Breaking Changes**
 
+  - Interface FhirService no longer has parameter accessPolicies
```

Comment:
could you point me the breaking?

Reply:
https://github.com/Azure/azure-rest-api-specs/pull/25583